ui/src/services/NodeVolumesUtils.test.js

```diff
@@ -420,3 +422,96 @@ it('should return error when called with failed Ready condition', () => {
 });
 
 // }}}
+
+// getVolumeListData
+const props_volume_filtered_by_node_master0 = {
+  history: {
+    length: 12,
+    action: 'POP',
+    location: {
+      pathname: '/volumes/prom-m0-reldev',
+      search: '?node=master-0',
+      hash: '',
+      key: 'u757uz',
+    },
+  },
+  location: {
+    pathname: '/volumes/prom-m0-reldev',
+    search: '?node=master-0',
+    hash: '',
+    key: 'u757uz',
+  },
+  match: {
+    path: '/volumes',
+    url: '/volumes',
+    isExact: false,
+    params: {},
+  },
+};
+
+it('should return the volume list', () => {
+  const result = getVolumeListData(
+    state_volume_filtered_by_node_master0,
+    props_volume_filtered_by_node_master0,
+  );
+  const volumelist_filtered_by_node_master0 = [
+    {
+      name: 'master-0-alertmanager',
+      node: 'master-0',
+      usage: '0.41',
+      status: 'link',
+      bound: 'Yes',
+      storageCapacity: '5Gi',
+      storageClass: 'metalk8s-prometheus',
```

Comment:
All occurrences of `metalk8s-prometheus` storage class should probably be replaced by `metalk8s` according to the following PR #2755

Reply:
Got it! Thanks for the sync up!